### PR TITLE
add CfA quiz instructions

### PR DESCRIPTION
**Pull request by zmx721**

### CODE REVIEW

File: resources_pages/quiz_cfa.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: MDS quizzes - CfA
+---
+
+
+## Accommodations
+
+If you have an accommodation letter from the Center for Accessibility, please send them to the course coordinators to ensure you receive your approved accommodations throughout the program.
```
Comment (phchen5) [inline resources_pages/quiz_cfa.md]:
```suggestion
If you have an accommodation letter from the Center for Accessibility (CfA), please send it to the course coordinators to ensure that your approved accommodations are provided throughout the program.
```

File: resources_pages/quiz_cfa.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+If **all** of your accommodations **can** be accommodated by the CBTF (check the list [here](https://cbtf.ubc.ca/students/accommodations)), you will still write your quizzes at the CBTF. Please follow instructions on [MDS quizzes](https://ubc-mds.github.io/resources_pages/quiz/).
```
Comment (phchen5) [inline resources_pages/quiz_cfa.md]:
```suggestion
If **all** of your accommodations **can** be met by the CBTF (check the list [here](https://cbtf.ubc.ca/students/accommodations)), you will continue to write your quizzes at the CBTF. Please follow the instructions on the [MDS quizzes](https://ubc-mds.github.io/resources_pages/quiz/) page to register for your quizzes.
```

File: resources_pages/quiz_cfa.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+Otherwise, please follow the instructions below.
```
Comment (phchen5) [inline resources_pages/quiz_cfa.md]:
```suggestion
For accommodations that cannot be provided by the CBTF, follow the instructions below. You should still go through the [MDS quiz policies](https://ubc-mds.github.io/resources_pages/quiz/). Just note that the following sections will not apply to you:
- [PrairieTest and Making a Reservation for your Quiz](https://ubc-mds.github.io/resources_pages/quiz/#prairietest-and-making-a-reservation-for-your-quiz)
- [Quiz Room Procedures](https://ubc-mds.github.io/resources_pages/quiz/#quiz-room-procedures)
```

File: resources_pages/quiz_cfa.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## List of MDS Quiz instructions that diverge
```
Comment (phchen5) [inline resources_pages/quiz_cfa.md]: Combined this section to the sentence ^^^

File: resources_pages/quiz_cfa.md [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+
+You should still go through the [MDS quizzes](https://ubc-mds.github.io/resources_pages/quiz/) page. The only sections that do not apply to you is:
+- [PrairieTest and Making a Reservation for your Quiz](https://ubc-mds.github.io/resources_pages/quiz/#prairietest-and-making-a-reservation-for-your-quiz)
+- [Quiz Room Procedures](https://ubc-mds.github.io/resources_pages/quiz/#quiz-room-procedures)
+
+
+## CfA Quiz Booking
+
+- On [PrairieTest](https://us.prairietest.com/pt), you do **not** need to make a reservation. Instead, we will assign you to a special session allowing you to write your quiz outside of the CBTF.
```
Comment (phchen5) [inline resources_pages/quiz_cfa.md]:
```suggestion
You do **not** need to make a reservation on [PrairieTest](https://us.prairietest.com/pt). Instead, we will assign you to a special session to write your quiz outside of the CBTF.
```

File: resources_pages/quiz_cfa.md [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+- You should book each of your quizzes with the CfA. Instructions are provided by [the CfA](https://students.ubc.ca/about-student-services/centre-for-accessibility).
```
Comment (phchen5) [inline resources_pages/quiz_cfa.md]:
```suggestion
- Book each of your quizzes through the CfA following their instructions [here](https://students.ubc.ca/about-student-services/centre-for-accessibility).
```

File: resources_pages/quiz_cfa.md [hunk 7]
```diff
@@ -0,0 +1,41 @@
...
+    - You can pick your own quiz times, as long as they fall in the quiz window specified in the calendar, **and** they cannot happen during your lecture and lab times.
```
Comment (phchen5) [inline resources_pages/quiz_cfa.md]:
```suggestion
    - You can choose your quiz times as long as they fall within the quiz window specified in the calendar **and** do not overlap with your lecture or lab times.
```

**Review by phchen5 — COMMENTED** [review]
lgtm!

File: resources_pages/quiz_cfa.md [hunk 8]
```diff
@@ -0,0 +1,41 @@
+---
+layout: page
+title: MDS quizzes - CfA
+---
+
+
```
Comment (phchen5) [inline resources_pages/quiz_cfa.md:7]:
```suggestion
## What is CfA?

The CfA stands for the Centre for Accessibility. It provides disability-related accommodations and programming designed to remove barriers for students with disabilities or ongoing medical conditions in all aspects of university life. For more information, visit the [Centre for Accessibility](https://students.ubc.ca/about-student-services/centre-for-accessibility).

In the MDS program, if you have a disability or ongoing medical condition, you can request quiz accommodations through the CfA.
```

Comment (phchen5) [reply resources_pages/quiz_cfa.md:7]: add a brief intro to CfA in the beginning

**Review by ZacWarham — APPROVED** [review]
Thanks Betty

**Pull request merged**